attestation-service/attestation-service/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [features]
-default = [ "rvps-grpc", "all-verifier" ]
+default = [ "rvps-grpc", "rvps-builtin", "all-verifier" ]
```

Comment:
Why we need enable both "rvps-grpc" and "rvps-builtin" as default feature?

Reply:
It is for compilation check and lint check. I will disable unused features in binary's Makefile.